pkg/restapi/issuer/operation/operations.go

```diff
@@ -1176,22 +1193,68 @@ func (o *Operation) storeUserInvitationMapping(mapping *UserInvitationMapping) e
 }
 
 // TODO fulfillment shouldn't be saved, need re-design [Issue#560]
-func (o *Operation) saveCredentialFulfillment(thID string, fulfillment *verifiable.Presentation) error {
-	fbytes, err := fulfillment.MarshalJSON()
+func (o *Operation) saveCredentialAttachmentData(thID string, attachments credentialAttachments) error {
+	manifestBytes, err := json.Marshal(attachments.ManifestID)
```

Comment:
Why json.marshal for string?

simply `[]byte(attachments.ManifestID)`

Reply:
also, name `credentialAttachments` is too generic, use something meaningful like `credentialOffered`, `interactionData` etc